packages/base/tsconfig.json

```
@@ -16,6 +16,7 @@
     "src/schema/*.json",
     "src/_interface/*.json",
     "src/*.json",
-    "src/types/*"
+    "src/types/*",
+    "src/**/*.json"
```

```suggestion
    "src/dialogs/symbology/components/color_ramp/cmocean.json"
```

`src/**/*.json` would obviate the need for several other lines in this list. I don't know for sure if there are other JSON files we _don't_ want to include. I think it's easiest to continue the pattern that already exists of enumerating specific json files or folders we want to include.